src/components/RengoManagementPane/RengoManagementPane.tsx

```diff
@@ -1,18 +1,18 @@
 /*
  * Copyright (C) 2012-2017  Online-Go.com
  *
- * This program is free software: you can redistribute it and/or modify
```

Comment:
I reckon we should retain the This's in the legal preamble :)

Reply:
Whoa, how on earth did I delete that line!!?

Reply:
Oh LOL ... I deleted all the "this" :D

Reply:
Good old replace all haha wHaT cOuLd Go WrOnG 🤪🫠😂